src/tooltip/index.tsx

```diff
@@ -45,10 +43,7 @@ export const TooltipTriggerButton = forwardRef<
             excludeFromTabOrder
             ref={ref}
         >
-            <Icon
```

Comment:
Why did we remove this? I don't think `TooltipTriggerButton` should be a general purpose wrapper around anything we want a tooltip on, then, if we try to use it on a pre-existing button in the portal, the resulting html will be something like:

```html
<button>
    <button>content</button>
</button>
```